Improvements to support Debian based servers (#65)
plone/cookieplone-templates: project/{{ cookiecutter.__folder_name }}/devops/README.md

Note: The `.env` file is included in `.gitignore`, ensuring environment-specific configurations aren't pushed to the repository.

### Server installation

> You need to Ubuntu or Debian in {{ cookiecutter.hostname }}, enable SSH and install python3 there.

Reviewer: This sentence still does not make sense, even after I cleaned up a part of it.

```suggestion
You need to Ubuntu or Debian in {{ cookiecutter.hostname }}, enable SSH, and install a supported version of Python 3 there.
```

Reviewer: Perhaps "You need to specify Ubuntu or Debian in..."?

And is "there" in the file `.env`?

I see `ansible_lsb.id` below. Does that figure into this?

Author: Yes, the `ansible_lsb.id` figures out if we are using Debian or Ubuntu in the server. No need to set it anywhere.

Reviewer: OK. How do we fix this sentence? It still doesn't make sense.

> You need to Ubuntu or Debian in {{ cookiecutter.hostname }}, enable SSH, and install a supported version of Python 3 there.

Reviewer: This still doesn't make sense:

"You need to Ubuntu or Debian in {{ cookiecutter.hostname }}"

Author: What about, `you need an Ubuntu or Debian based system in {{cookiecutter.hostname}}`

Reviewer: ```suggestion
You need either a Ubuntu or Debian based system for {{ cookiecutter.hostname }}, enable SSH, and install a supported version of Python 3 on that system.
```

Reviewer: I think this does it. This is good.
